src/index.js

```diff
@@ -12,6 +12,12 @@ import LineStream from './line-stream';
 import ParseStream from './parse-stream';
 import Parser from './parser';
 
+export {
```

Comment:
`module.exports` is probably the recommended way at module boundaries.
Though, this shouldn't affect us because babel does grab the `default` export by default.